app/controllers/admin/configs_controller.rb

```diff
@@ -129,10 +129,23 @@ class ConfigsController < Admin::ApplicationController
         home_feed_minimum_score
       ].freeze
 
+    VALID_IMAGE_URLS =
```

Comment:
I feel like `VALID_IMAGE_URLS` is not quite describing what you want here. Maybe `VALID_IMAGE_FIELDS` or just `IMAGE_FIELDS`?